src/backend/vulkan/src/lib.rs

```diff
@@ -1070,4 +1351,8 @@ impl hal::Backend for Backend {
     type Semaphore = native::Semaphore;
     type Event = native::Event;
     type QueryPool = native::QueryPool;
+
+    type Display = native::Display;
+    type Plane = u32;
```

Comment:
why does this need to be abstracted away? Can't we just say at the gfx-hal level that `type Plane = u32`, and be done with it?

Reply:
I thought that others future apis could implement this path. Drm/kms and vulkan use a u32 but maybe another api could use an opaque object as handle. I have simply thought that representing it in that way could be be more future proof, but currently there is no need to be abstracted, so i can remove it and simply use u32 if you desire.

Reply:
We can cross this bridge when we get there. Unless we have exact knowledge of the future work, let's stick to the minimum of changes here.